CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ The format is based on [Keep a Changelog], and this project adheres to
 [Keep a Changelog]: https://keepachangelog.com/en/1.0.0/
 [Semantic Versioning]: https://semver.org/spec/v2.0.0.html
 
+## [Unreleased]
+
+- Removed redundant marshalling utility functions `MarshalMessage()`,
+  `UnmarshalMessage()`, `MustMarshalMessage()`, and `MustUnmarshalMessage()`
```

Comment:
```suggestion
### Removed

- **[BC]** Remove `MarshalMessage()`, `UnmarshalMessage()`, `MustMarshalMessage()`, and `MustUnmarshalMessage()`
```

These changelogs are organised in to different sections based on the guidelines at https://keepachangelog.com/en/1.0.0/.  In addition to that I add the `**[BC]**` marker to draw attention to things that will break when upgrading.